clicommand/pipeline_upload.go

```diff
@@ -275,11 +284,33 @@ var PipelineUploadCommand = cli.Command{
 			searchForSecrets(l, &cfg, environ, result, src)
 		}
 
-		if cfg.JWKSFile != "" {
-			key, err := jwkutil.LoadKey(cfg.JWKSFile, cfg.JWKSKeyID)
+		var (
+			key signature.Key
+		)
+
+		switch {
+		case cfg.JWKSKMSKeyID != "":
+			awscfg, err := config.LoadDefaultConfig(
```

Comment:
There's a small risk that this could resolve credentials in a different order to the how the rest of the agent does it.
But since this is a new feature, and we need to move to this way soon (when v1 of the SDK gets deprecated), I think this is a good place to start using this approach